source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRatingInputRenderer.mm

```diff
@@ -0,0 +1,91 @@
+//
+//  ACRRatingInputRenderer.m
+//  AdaptiveCards
+//
+//  Created by Abhishek on 14/05/24.
+//  Copyright © 2024 Microsoft. All rights reserved.
+//
+
+#import "ACRInputNumberRenderer.h"
+#import "ACOBaseCardElementPrivate.h"
+#import "ACOBundle.h"
+#import "ACOHostConfigPrivate.h"
+#import "ACRContentHoldingUIView.h"
+#import "ACRInputLabelViewPrivate.h"
+#import "ACRNumericTextField.h"
+#import "ACRTextInputHandler.h"
+#import "ACRRatingInputRenderer.h"
+#import "UtiliOS.h"
+#import "RatingInput.h"
+#import "ACRRatingView.h"
+#import "ACRRatingInputDataSource.h"
+
+@implementation ACRRatingInputRenderer
+
++ (ACRRatingInputRenderer *)getInstance
+{
+    static ACRRatingInputRenderer *singletonInstance = [[self alloc] init];
+    return singletonInstance;
+}
+
++ (ACRCardElementType)elemType
+{
+    return ACRRatingInput;
+}
+
+- (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
+           rootView:(ACRView *)rootView
+             inputs:(NSMutableArray *)inputs
+    baseCardElement:(ACOBaseCardElement *)acoElem
+         hostConfig:(ACOHostConfig *)acoConfig;
+{
+    std::shared_ptr<HostConfig> config = [acoConfig getHostConfig];
+    std::shared_ptr<BaseCardElement> elem = [acoElem element];
+    std::shared_ptr<RatingInput> ratingInput = std::dynamic_pointer_cast<RatingInput>(elem);
+    
+    NSInteger maxValue = ratingInput->GetMax() > 5 ? 5: ratingInput->GetMax();
```

Comment:
Can you put the "5" as a constant instead of hardcoded?

Reply:
I couldn't find any constants file in codebase. Literal values like these are used everywhere in code. Do you want me to create a new constants file for this?

Reply:
You are right. Then is ok as it is.